### ISSUE TITLE
Towne needs adjustment on  Invoice  Print

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#197]
In GitLab by @smasih on Jan 12, 2017, 23:42

From: "Deb Crown" <dcrown@towneanswering.com>

Date: 12-Jan-2017 22:12

Subject: font changes

To: "Support" <support@sabilling.com>

Cc: "Charlie Crown" <ccrown@towneanswering.com>

|

For some reason, the font or the spacing or the placement of the type has changed within SA.

The last time we generated invoices, the top line of the address barely made it into the window of our envelopes.

We are not sure why this will be the 3rd time our invoices have to be adjusted.

|

I believe the type needs to be lowered one – two lines. If you need to work with someone here,  Charlie will be the person to work with on this – please email or call him to set up an appointment to make the necessary adjustments.

 
|

|

 

Thank you,

 

|

Deb Crown
![image001](/uploads/e267b9cf6d0b4637c371d66eb06760c4/image001.jpg)

215-723-4316

www.TowneAnswering.com

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jan 13, 2017, 01:33

@tim.traylor @abakshi @kdjstudios 
Hi all,

Embedded a sample Invoice image with numbers on each section of it. **So that they can easily point out which section is needed to be re-aligned.**

![invoice_TWN-31548](/uploads/455ad7a55ee7f7eb5eab265580a1f1fe/invoice_TWN-31548.jpg)

**I assumed it to be section 3 and made the changes**

Lowered section 3 by 10pt

**Before** [invoice_TWN-31548.pdf](/uploads/47779ebf4f643a6256f094372a0da443/invoice_TWN-31548.pdf)

**After** [towne_final.pdf](/uploads/1cce3483ed35315ef60284af4a0a9713/towne_final.pdf)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 18, 2017, 08:42

I have just emailed Deb Crown, and am asking for an update of which section is to be moved and roughly how far to move it. I will follow up with her response.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 18, 2017, 10:49

@smasih 
I recieved the following from Deb:
Thanks for working on this – the adjustment you made for section 3 is definitely helpful. Could you lower it the same amount again? (10pts?) 
Our envelope window is large enough to accommodate that adjustment.
Also, could you lower section 5 by the same amount (20pts??) 
I believe everything else is fine….
If you could send me the final adjusted template, I can print and check for sure.
Thanks again!

- In summary Section 3 and Section 5 will both need to be lowed a total of 20 pt from the original. Please attach the adjusted template once completed. Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jan 19, 2017, 04:51

@abakshi 

Hi @kdjstudios ,

Thanks for updating on this, I have attached a sample invoice for on of the Towne's account.

[invoice_TWN-31548.pdf](/uploads/d0a026b5bd8fe137d23e158deb38a174/invoice_TWN-31548.pdf)

Please let us know if there are any further amendments required, and once its done we will deploy changes on Hosted Production only for **Towne Site**

**Comment by kdjstudios** [comment]
In GitLab by @abakshi on Jan 19, 2017, 05:32

Changed title: **Towne needs a{-n-}justment on  Invoice  Print** → **Towne needs a{+d+}justment on  Invoice  Print**

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 19, 2017, 14:39

Deb has confirmed that the changes made in the sample invoice above are satisfactory and are ready for production.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jan 20, 2017, 01:37

Changes have been tested and we will be releasing it tomorrow morning (January 21st, 2017 our time IST ).

Please note **this change is exclusive for Towne, so rest would be same.**

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 20, 2017, 08:34

@smasih Thank you, I will update the client at end of day.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jan 21, 2017, 04:52

Hi @kdjstudios, just an update that changes are live, but only for towne.

thanks

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 23, 2017, 08:44

Thanks @smasih I have informed Deb to confirm everything is still matching and satisfactory. Thanks!

**Comment by kdjstudios** [comment]
In GitLab by @abakshi on Jan 24, 2017, 03:35

Already released into Answernet production.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 6, 2017, 16:30

Closing ticket, this was completed.
